### PR TITLE
python312Packages: aiohttp: 3.11.2 -> 3.11.9, yarl 1.17.3 -> 1.18.3, cryptography 43.0.1 -> 44.0.0, pyopenssl 24.2.1 -> 24.3.0

### PR DESCRIPTION
**Pull request by mweinelt**
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.3
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.4
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.5
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.6
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.7
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.8
https://github.com/aio-libs/aiohttp/releases/tag/v3.11.9

https://github.com/bdraco/aiohappyeyeballs/blob/v2.4.4/CHANGELOG.md
https://github.com/aio-libs/propcache/blob/refs/tags/v0.2.1/CHANGES.rst
https://github.com/aio-libs/yarl/blob/v1.18.3/CHANGES.rst

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/yarl/default.nix [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 buildPythonPackage rec {
   pname = "yarl";
-  version = "1.17.1";
+  version = "1.18.3";
 
   disabled = pythonOlder "3.8";
```
Comment (dotlambda) [inline pkgs/development/python-modules/yarl/default.nix:23]:
```suggestion
  disabled = pythonOlder "3.9";
```

File: pkgs/development/python-modules/pyjwt/default.nix [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 buildPythonPackage rec {
   pname = "pyjwt";
-  version = "2.9.0";
+  version = "2.10.1";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```
Comment (dotlambda) [inline pkgs/development/python-modules/pyjwt/default.nix:20]:
```suggestion
  disabled = pythonOlder "3.9";
```

File: pkgs/development/python-modules/aiohttp/default.nix [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
 buildPythonPackage rec {
   pname = "aiohttp";
-  version = "3.11.2";
+  version = "3.11.9";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```
Comment (dotlambda) [inline pkgs/development/python-modules/aiohttp/default.nix:52]:
```suggestion
  disabled = pythonOlder "3.9";
```

**Review by dotlambda — COMMENTED** [review]
Feel free to remove the `disabled` attributes instead.

**Comment by mweinelt** [comment]
Removed disabled as suggested.

**Pull request merged**